diff --git a/dist/compile-D4HwBWGK.mjs b/dist/compile-D4HwBWGK.mjs
index 1d7ba062f39bd22e11d5161d9d15c2ed4841485f..4fce0e64d2aad55a8ee2d495e684db9584c73b42 100644
--- a/dist/compile-D4HwBWGK.mjs
+++ b/dist/compile-D4HwBWGK.mjs
@@ -7,6 +7,7 @@ import attrs from "markdown-it-attrs";
 import { full } from "markdown-it-emoji";
 import { bundledLanguages, createHighlighter, isSpecialLang } from "shiki";
 import container from "markdown-it-container";
+import footnote from "markdown-it-footnote";
 //#region src/plugins/containers.ts
 function createContainer(md, klass, defaultTitle) {
 	return [
@@ -304,6 +305,7 @@ async function createCompiler(options) {
 	markdown.core.ruler.after("block", "stash_fence_info", stashFenceInfo);
 	markdown.use(attrs);
 	markdown.use(full);
+	markdown.use(footnote);
 	markdown.use(anchor, { permalink: anchor.permalink.ariaHidden({ placement: "before" }) });
 	markdown.use(tocPlugin);
 	return markdown;
@@ -328,10 +330,9 @@ function extractHeadings(tokens) {
 async function compile(source, options) {
 	const { data: frontmatter, content } = matter(source);
 	const compiler = await getCompiler(options);
-	const env = {};
-	const headings = extractHeadings(compiler.parse(content, env));
+	const headings = extractHeadings(compiler.parse(content, {}));
 	return {
-		html: await compiler.renderAsync(content, env),
+		html: await compiler.renderAsync(content, {}),
 		frontmatter,
 		headings,
 		title: frontmatter.title ?? headings.find((heading) => heading.depth === 1)?.text ?? ""
